.github/pull_request_template.md

```diff
@@ -1,3 +1,23 @@
-<!-- Please add the `ready-to-merge` label when you have received both approvals instead of clicking the merge button
-Using the `ready-to-merge` label adds your approved PR to the merge train and when master is in a green state from the previous merge.
-Mergify will merge your PR based on the queue -->
+<!--
+Please add the `ready-to-merge` label when the pull request has received the appropriate approvals.
+Using the `ready-to-merge` label adds your approved pull request to the merge queue where it waits to be merged.
+Mergify will merge your pull request based on the queue assuming your pull request is still in a green state after the previous merge.
+
+What to do when the `ready-to-merge` label not working:
+
+- Do you have two approvals?
+  - At least two approvals are required in order to merge to the master branch.
+- Are there any reviewers that are still requested for review?
+  - If the pull request has received the necessary approvals, remove any additional reviewer requests that are pending.
+    - e.g.
+      - Three reviewers were added comments but you already have two necessary approvals and the third reviewer's comments are no longer applicable. You can remove the third person as a reviewer or have them approve the pull request.
+      - A team was added as a reviewer because of a change to a file but the file change has been undone. At this point, it should be safe to remove the team as a reviewer.
+- Are there any other pull requests in the merge queue before?
+  - Mergify handles the queueing, your pull request will eventually get merged.
+
+When to contact someone for assistance when trying to merge via `ready-to-merge` label?
```

Comment:
could change this to use a colon instead of question mark so it's similar to the section above:
```suggestion
When to contact someone for assistance when trying to merge via `ready-to-merge` label:
```

or change the above section (``What to do when the `ready-to-merge` [...]``) to use a question mark